README-dev.md

```diff
@@ -4,23 +4,23 @@
 
 Many packages depend on other packages within the monorepo. When this happens, follow these rules:
 
-  1. All packages must use **master version** of sibling packages.
-  2. Exception to (1) are packages that represent a GAE/firebase app which must use the last published version.
-  3. To differentiate published vs unpublished version. Master version (in package.json) must end with suffix `-dev` and should not be published.
-  4. If a developer want to publish a version; then after publishing it needs to set master version to next `-dev` version and change all package.json that require on it.
+1.  All packages must use **master version** of sibling packages.
```

Comment:
Were the changes to the readmes intentional? If not, please revert.

Reply:
I wonder where this came from...